src/app/project-links.ts

```diff
@@ -4,16 +4,15 @@ import { Injectable } from "@angular/core";
 export default class ProjectLinks {
     public static urlHome = "/";
     public static urlProfile = "profile";
-    public static urlLogin = "login";
+    public static urlLogin = "v/login";
     public static urlGithubCallback = "github_callback";
```

Comment:
Maybe this callback should also be "v/..." prefixed ?

Reply:
Yes, I agree